mbed_lstools/lstools_base.py

```diff
@@ -28,7 +28,7 @@
 class MbedLsToolsBase:
     """ Base class for mbed-lstools, defines mbed-ls tools interface for mbed-enabled devices detection for various hosts
     """
-    def __init__(self):
+    def __init__(self, **kwargs):
```

Comment:
It looks like `skip_retarget` is the only keyword argument that's being used right now, so why not make it explicit here?

```python
def __init__(self, skip_retarget=False):
```